### PR TITLE
New version: SAGAUserGroupAssociation.SAGAGIS version 9.1.1

### PR DESCRIPTION
**Pull request by xyzallan**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/115283&drop=dogfoodAlpha

### CODE REVIEW

**Comment by xyzallan** [comment]
Microsoft Defender SmartScreen prevents the silent install. How to bypass it?

**Comment by stephengillie** [comment]
> Microsoft Defender SmartScreen prevents the silent install. How to bypass it?

Automatic Validation bypasses the SmartScreen prompt. Manual Validation waits for me to click through it.

File: manifests/s/SAGAUserGroupAssociation/SAGAGIS/9.1.1/SAGAUserGroupAssociation.SAGAGIS.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- sprj
+Installers:
+- Architecture: x64
+  InstallerUrl: https://deac-ams.dl.sourceforge.net/project/saga-gis/SAGA%20-%209/SAGA%20-%209.1.1/saga-9.1.1_x64_setup.exe
+  InstallerSha256: CBFD6D991DF3B7C59C9934178D64D5FB0C42B2ACF25713C33A7CABB7C002958F
+  ProductCode: SAGA - System for Automated Geoscientific Analyses_is1
+- Architecture: x86
+  InstallerUrl: https://deac-riga.dl.sourceforge.net/project/saga-gis/SAGA%20-%209/SAGA%20-%209.1.1/saga-9.1.1_x64_setup.exe
+  InstallerSha256: CBFD6D991DF3B7C59C9934178D64D5FB0C42B2ACF25713C33A7CABB7C002958F
+  ProductCode: SAGA - System for Automated Geoscientific Analyses_is1
```
Comment (stephengillie) [inline manifests/s/SAGAUserGroupAssociation/SAGAGIS/9.1.1/SAGAUserGroupAssociation.SAGAGIS.installer.yaml:31]:
```suggestion
```

Comment (stephengillie) [reply manifests/s/SAGAUserGroupAssociation/SAGAGIS/9.1.1/SAGAUserGroupAssociation.SAGAGIS.installer.yaml:31]: Hi @xyzallan,

This appears to be a 64-bit only installer. Is that accurate?

Comment (xyzallan) [reply manifests/s/SAGAUserGroupAssociation/SAGAGIS/9.1.1/SAGAUserGroupAssociation.SAGAGIS.installer.yaml:31]: It seems so yes.

**Comment by stephengillie** [comment]
Hi @xyzallan,

This package has passed validation, and can be merged into the repo. Just click "Ready for review" to convert to an open PR.

**Review by ItzLevvie — APPROVED** [review]

**Comment by ItzLevvie** [comment]
Close with reason: Stale pull request;

**Pull request closed without merging**